src/addons/unicode-base.js

```diff
@@ -224,4 +224,20 @@ module.exports = function(XRegExp) {
         XRegExp.cache.flush('patterns');
     };
 
+    /**
+     * Test if the given name is a legal Unicode Slug for use in XRegExp `\p` or `\P` regex constructs.
+     *
+     * @memberOf XRegExp
+     * @param {String} name Name by which the Unicode slug may be recognized (case-insensitive),
+     *   e.g. `'N'` or `'Number'`.
+     *   
+     *   The given name is matched against all registered Unicode names and aliases.
+     *
+     * @return {Object} Truthy when the name matches a Unicode slug (the internal slug definition
+     *   object is returned); `false` when the name does not match *any* Unicode name or alias. 
+     */
+    XRegExp.isUnicodeSlug = function(name) {
```

Comment:
Variables/args named `slug` in Unicode Base have always *already* had their value passed through `normalize`. In other words it's lowercase and has had any hyphens, spaces, and underscores removed. It's a unique value that represents all the valid variations of a particular Unicode property name (not including explicit aliases). Therefore IMO the word "slug" isn't appropriate in this method's name.

Also, I'm not a fan of this API that returns a truthy or boolean value. Based on the name this should always return a boolean. If you need to get back the data for the Unicode property, that's fine, but I don't think this should be an official/documented method. Recommend changing the method to something like:

```js
// Intentionally undocumented
XRegExp._getUnicodeData = function(name) {
  var slug = normalize(name);
  return unicode[slug];
}
```